crates/tables/src/lib.rs

```diff
@@ -82,6 +82,9 @@ tables!(
         // HMAC-256 keys for this data-plane.
         // The first is used for signing, and any key may validate.
         val hmac_keys: Vec<String>,
+        // HMAC-256 keys for this data-plane in sops-encrypted yaml document format
+        // The first is used for signing, and any key may validate.
+        val encrypted_hmac_keys: String,
```

Comment:
nit: `models::RawValue` ? Basically a String, but enforces / clarifies that it's JSON, and is conventional here